atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleOverflowWriteTable.java

```diff
@@ -80,7 +83,7 @@ public void put(Collection<Map.Entry<Cell, byte[]>> data, long ts) {
         for (Entry<Cell, byte[]> entry : data) {
             Cell cell = entry.getKey();
             byte[] val = entry.getValue();
-            if (val.length <= 2000) {
+            if (val.length <= OVERFLOW_THRESHOLD) {
```

Comment:
`ORACLE_ OVERFLOW_THRESHOLD` to `AtlasDBConstants` since its used in multiple files. Thanks for extracting it!